<launch>
  <rosparam file="$(find objsearch_toplevel)/config/global_params.yaml" command="load" />

  <!-- These values are used to further adjust the floor and ceiling z-value
       which is used to remove points which make up those parts of the cloud -->
  <arg name="ceiling_offset" default="0.2" />
  <param name="/rotationtest/ceiling_offset" value="$(arg ceiling_offset)" />
  <arg name="floor_offset" default="0.2" />
  <param name="/rotationtest/floor_offset" value="$(arg floor_offset)" />
  <arg name="cloud_dir" />
  <param name="/rotationtest/cloud_dir" value="$(arg cloud_dir)" />

  <node name="readtest" pkg="preprocess" type="readtest" output="screen" />
</launch>
